sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>This is the legacy Azure Event Hubs .NET Standard client library. To view releases for the current generation of the Azure client libraries for .NET, please see: https://aka.ms/azsdk/releases.  For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>4.4.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.EventHubs is available as of February 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

```suggestion
```suggestion
    <Description>Please be aware that a newer package, Azure.Messaging.EventHubs, was released in February 2020.  While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development. Read the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

---

Keeping current text as concluded in https://github.com/Azure/azure-sdk-for-net/pull/20402#discussion_r614228896